lib/AcqList/hooks/useFilters.js

```diff
@@ -38,7 +38,7 @@ const useFilters = (resetData, initialFilters = {}) => {
   );
 
   const changeSearch = useCallback(
-    e => setSearchQuery(e.target.value?.trim()),
+    e => setSearchQuery(e.target.value),
```

Comment:
hm, after this change we'll have issues with spaces, I don't think it's correct fix

Reply:
@NikitaSedyx - would it be acceptable to pass a flag to `useFilters` to be able to customize the trimming?